gk/main.c

```diff
@@ -386,33 +434,77 @@ setup_gk_instance(unsigned int lcore_id, struct gk_config *gk_conf)
 	return ret;
 }
 
+static struct flow_entry *
+add_new_flow_from_policy(
```

Comment:
Given that this function doesn't fully initialize the new flow entry, please add a comment what the function does.